tools/update-tester/src/installer.rs

```diff
@@ -18,10 +18,11 @@ pub fn install_all_versions(
     reinstall: &HashSet<&str>,
 ) -> xshell::Result<()> {
     let extension_dir = path!(root_dir / "extension");
-    let install_toolkit = || -> xshell::Result<()> {
-        let _d = pushd(&extension_dir)?;
+    let install_toolkit = |pgx_version| -> xshell::Result<()> {
         let _e = pushenv("CARGO_TARGET_DIR", "../target/extension");
-        quietly_run(cmd!("cargo pgx install -c {pg_config}"))
+        quietly_run(cmd!(
+            "/pgx-{pgx_version}/bin/cargo-pgx pgx install -c {pg_config}"
```

Comment:
A downside of this approach is likely not being able to run the update tester locally before pushing. I'm not sure whether that's currently part of anyone's workflow, but it would be broken until you installed `pgx` into those same locations on your machine.

Reply:
🤔

Reply:
That used to be part of my workflow on occasion when I needed to debug update issues, but really that only happened when debugging the update scripts or the meta-tests, which honestly wasn't all that frequent.

I think the way to support local builds would be to let the user pass in the root dir in which the `pgx-{pgx_version}`s reside. But I think we can leave that for a followup PR if @WireBaron and @epgts find it acceptable (maybe the next person to break updates should fix it 😈)

Reply:
I made running all the tests easy with `tools/build` specifically so that we COULD run them all early and often! And I do :)

That said, I don't think installing a couple versions of pgx into a special location is so onerous (assuming that is in fact all that's necessary; I had other problems, see below). Just please document that in the README.

Thanks!

Reply:
I see what's wrong; have a look at commit dbd10eb4357ffd6e039ce798d5a8824b7faa1956 .

Still would be nice to fix that error handling...

Reply:
I'd been unaware/forgotten about `tools/build`! I think rather than breaking it, I'd like to see it keep working and start using it myself 😜

I'm open to other solutions still, but assuming `pgx`s are installed at `/pgx/0.X` is the best I can come up with at the moment, so good to know it isn't too onerous. Duly noted about making others aware in the README.

From at your commit:
> Also we don't need to read and parse Cargo.toml repeatedly.
 
Don't we? Is there another (better) way to know which `cargo pgx` we want to call? Where the `pgx` we want will depend on which release we have checked out

Reply:
Hah, of course you're right, we do! My bad. I think something like the scoping changes are required though.